### DISCUSSION TITLE
Image preview failed when many images in one document

### DISCUSSION THREAD

**Discussion by NangeGao** [outline/outline#6233] [General]
When I insert about 60 images in one document, some images preview failed. I think this may be a bug. I find that every image has a related `<dialog>` element in source code, which is actually the preview modal. As I have 60 images in document, but the dialog element is only about 30 or 40. I refresh the page, the amount would change. 
Because of this, some images have no related dialog elements and will disappear once the mouse is clicked. When the mouse is hovered on a normal image, the cursor will show `zoom-in` style. For images without related dialog, the default mouse style is used.

![image](https://github.com/outline/outline/assets/6187897/e82c756f-5443-445e-8666-199b61294016)

**Comment by tommoor** [comment]
Could you use the bug template, which version of the software etc?

**Reply by NangeGao** [reply]
https://github.com/outline/outline/issues/6243 That's it. Thank you.
